### DISCUSSION TITLE
Can't populate modal select with pivot table

### DISCUSSION THREAD

**Discussion by ramiropiro** [filamentphp/filament#5293] [General]
I'm tring to fill a select with a pivot table on a modal before redirect to a edit page, but nothig happends, I tryied with a textfield and [object Object] shows up, any advices?

![image](https://user-images.githubusercontent.com/111913304/208658053-6de9bddf-6f24-4607-837a-4af4e8337271.png)
![image](https://user-images.githubusercontent.com/111913304/208658102-2252e665-9fda-4efd-a363-5da827e65e2b.png)
![image](https://user-images.githubusercontent.com/111913304/208658159-cd4c78f7-757d-4dcd-b0a6-6bccb289066e.png)

**Comment by danharrin** [comment]
`'modalCargos' => $record->cargos()->pluck('id')`

Make sure to pass `->options()` to the select too

**Reply by ramiropiro** [reply]
great!, now I have the select populated with all items, and the one in the pivot, "selected", is there a way that i only populate the select with the ones in the pivot and not all the options?

**Reply by danharrin** [reply]
have you investigated `$record->pivot`?
